### DISCUSSION TITLE
Generating React component props list against Js Doc

### DISCUSSION THREAD

**Discussion by alexf2** [teambit/bit#5659] [Help] [answered]
I wonder if there is a setting to make Bit extracting React functional component props Js Doc instead of propTypes?
By default Bit renders a component information page against propTypes and gets related JsDoc prop-descriptions if there any at propTypes. It does not read component's Js Doc and that is inconvenient as we need to duplicate JsDoc @param descriptions.

**Comment by itaymendel** [comment, marked as answer]
We are working on this implementation, it will take a little longer to complete - https://github.com/teambit/bit/pull/5657

**Reply by alexf2** [reply]
Thanks, Itay! That's great!
